spec/features/change_profile_spec.rb

```diff
@@ -30,8 +30,10 @@
         describe 'updated profile' do
           subject { user.reload }
 
-          it { expect(subject.full_name).to eq full_name }
-          it { expect(subject.profile.phone_number).to eq phone_number }
+          it do
```

Comment:
@dissident add description to `it`. I will not write it everywhere but it should be fixed.